src/components/o-testimonials/js/testimonials.js

```diff
@@ -0,0 +1,178 @@
+import Swipe from '../../../js/swipe';
+import on from '../../../js/on';
+import getAttribute from '../../../js/get-attribute';
+import UiEvents, { AXA_EVENTS, EVENTS } from '../../../js/ui-events';
+
+class Testimonials extends UiEvents {
+  static DEFAULTS = {
+    controlLeft: '.js-o-testimonials__control-left',
+    controlRight: '.js-o-testimonials__control-right',
+    slides: '.js-o-testimonial__item',
+    slider: '.js-o-testimonials',
+    navigator: '.js-o-testimonials__navigator',
+    animationLeft: 'o-testimonials__item_animation_left',
+    animationRight: 'o-testimonials__item_animation_right',
+    autoRotateDisabled: 'auto-rotate-disabled',
+    autoRotateTime: 'auto-rotate-time',
+  };
+
+  constructor(wcNode, options = {
+  }) {
+    // eslint-disable-next-line no-param-reassign
+    options = {
+      ...Testimonials.DEFAULTS,
+      ...options,
+    };
+
+    super(wcNode, options);
+
+    this.slideIndex = 0;
+    this.options = options;
+    this.wcNode = wcNode;
+
+    this.init();
+  }
+
+  init() {
+    this.slides = this.wcNode.querySelectorAll(this.options.slides);
+    this.controlLeft = this.wcNode.querySelector(this.options.controlLeft);
+    this.controlRight = this.wcNode.querySelector(this.options.controlRight);
+    this.slider = this.wcNode.querySelector(this.options.slider);
+    this.autoRotateDisabled = getAttribute(this.wcNode, this.options.autoRotateDisabled);
```

Comment:
`getAttributes` would give you a hash of all attributes available, but this does not change workability.